src/System.CommandLine/System.Runtime.CompilerServices/Range.cs

```diff
@@ -8,23 +8,23 @@
 
 namespace System
 {
-    /// <summary>Represent a type can be used to index a collection either from the start or the end.</summary>
+    /// <summary>Represents a type that can be used to index a collection either from the beginning or the end.</summary>
     /// <remarks>
-    /// Index is used by the C# compiler to support the new index syntax
-    /// <code>
-    /// int[] someArray = new int[5] { 1, 2, 3, 4, 5 } ;
+    /// <see cref="T:System.Index" /> is used by the C# compiler to support the <c>>^</c> or ["index from end" operator](https://learn.microsoft.com/dotnet/csharp/language-reference/operators/member-access-operators#index-from-end-operator-):
+    /// <code language="csharp">
+    /// int[] someArray = new int[5] { 1, 2, 3, 4, 5 };
     /// int lastElement = someArray[^1]; // lastElement = 5
     /// </code>
     /// </remarks>
     internal readonly struct Index : IEquatable<Index>
```

Comment:
Because Index and Range are internal types here, and .NET documents the corresponding types in the System namespace, I don't think it is necessary to spend much time on improving their API docs.